libfwupdplugin/fu-device.c

```diff
@@ -5555,6 +5557,21 @@ fu_device_security_attr_new(FuDevice *self, const gchar *appstream_id)
 	attr = fu_security_attr_new(priv->ctx, appstream_id);
 	fwupd_security_attr_set_plugin(attr, fu_device_get_plugin(FU_DEVICE(self)));
 	fwupd_security_attr_add_guids(attr, fu_device_get_guids(FU_DEVICE(self)));
+
+	/* if the device has a parent of main-system-firmware then add those GUIDs too */
+	if (fu_device_has_parent_guid(self, FU_DEVICE_GUID_MAIN_SYSTEM_FIRMWARE)) {
```

Comment:
This is going to match a lot of things, are you sure about it?

Reply:
Less than you think; grep for fu_device_security_attr_new

Reply:
Oh, I see.. OK.